version1/04 - Web services.md

```diff
@@ -836,8 +836,8 @@ Content-Type: application/json
           "fieldId": string,
           "fieldName": string,
           "fieldDescription": string,
-          "fieldType": string|long|double|timestamp,
-          "fieldValues": [string|long|double|timestamp]
+          "fieldType": string|long|double|timestamp|encrypted,
+          "fieldValues": [string|long|double|timestamp|encrypted]
```

Comment:
As users can't create pre-defined encrypted values, this line should be reverted.

Reply:
Fixed in be13583bd7c237b563a43db275fa59dfdc1f2385.